plugins/app/report.api.md

```diff
@@ -232,12 +232,26 @@ const appPlugin: OverridableFrontendPlugin<
       ) => ExtensionBlueprintParams<AnyApiFactory>;
     }>;
     'api:app/app-language': OverridableExtensionDefinition<{
+      config: {
+        defaultLanguage: string | undefined;
+        availableLanguages: string[] | undefined;
+      };
+      configInput: {
+        defaultLanguage?: string | undefined;
+        availableLanguages?: string[] | undefined;
+      };
+      output: ExtensionDataRef<AnyApiFactory, 'core.api.factory', {}>;
+      inputs: {
```

Comment:
Oof, I don't like these types. I made https://github.com/backstage/backstage/pull/31992 to make things nicer